Query/ConditionManager.php

```diff
@@ -54,27 +54,27 @@ public function offsetSet($offset, $value)
         }
     }
 
-    public function offsetUnset($offset) {
+    public function offsetUnset($offset): void {
         unset($this->conditions[$offset]);
     }
 
-    public function rewind() {
-        return reset($this->conditions);
+    public function rewind(): void {
+        reset($this->conditions);
     }
 
-    public function current() {
+    public function current(): mixed {
         return current($this->conditions);
     }
 
-    public function key() {
+    public function key(): mixed {
         return key($this->conditions);
     }
 
-    public function next() {
-        return next($this->conditions);
+    public function next(): void {
+        next($this->conditions);
```

Comment:
What is the reason of this change? [Docs](https://www.php.net/manual/en/function.next.php) say:

> Returns the array value in the next place that's pointed to by the internal array pointer, or false if there are no more elements.